crate/server/src/config/command_line/jwt_auth_config.rs

```diff
@@ -9,28 +9,36 @@ use serde::{Deserialize, Serialize};
 pub struct JwtAuthConfig {
     /// The issuer URI of the JWT token
     ///
+    /// To handle multiple identity managers, add different parameters under each argument
+    /// (jwt-issuer-uri, jwks-uri and optionally jwt-audience), keeping them in
```

Comment:
sorry I find it hard to understand: what do we expect? please give 2 examples:
- 1 example with 1 single URI
- 1 example with several URIs
```suggestion
    /// jwt-issuer-uri, jwks-uri and optionally jwt-audience keeping them in
```